lynxkite-core/src/lynxkite/core/ops.py

```diff
@@ -239,12 +244,12 @@ def compute_id(self):
 def op(
     env: str,
     *names: str,
-    view="basic",
-    outputs=None,
-    params=None,
-    slow=False,
-    color=None,
-    cache=None,
+    view: str = "basic",
+    outputs: list[str] | None = None,
+    params: list[Parameter] | None = None,
+    slow: bool = False,
+    color: str | None = None,
+    cache: bool | None = None,
```

Comment:
To address Griffe warnings, which I saw when I checked if `mkdocs` still works. (It does.)